tests/inspection/test_info.py

```diff
@@ -348,24 +325,6 @@ def test_info_setup_missing_mandatory_should_trigger_pep517(
     assert spy.call_count == 1
 
 
-@pytest.mark.network
-def test_info_setup_missing_install_requires_is_fine(
```

Comment:
Same as above, we still need to test this case correct?

Reply:
similarly: this testcase seems to be aimed at a specific case within the setup reader, and that code path no longer exists.

Reply:
haha,  I say "seems to be" but now remember that I added it myself not a month ago at #9019 

I confirm that this testcase was aimed at a specific setup-reader scenario and is no longer interesting

Reply:
I agree the test case might not be interesting anymore as we do not really validate anything but that it "ran".